src/legacy/core_plugins/kibana/public/visualize/_index.scss

```diff
@@ -5,7 +5,6 @@
 //   visChart__legend--small
 //   visChart__legend-isLoading
 
-@import './editor/index';
-@import './embeddable/index';
-@import './listing/index';
-@import './wizard/index';
+@import 'np_ready/editor/index';
+@import 'np_ready/listing/index';
+@import 'np_ready/wizard/index';
```

Comment:
Another opportunity to not list `np_ready` multiple times

Reply:
Thanks, took same approach as in discover